packages/ffe-searchable-dropdown-react/src/SearchableDropdown.js

```diff
@@ -328,21 +353,27 @@ const SearchableDropdown = ({
                                     </Paragraph>
                                 </div>
                             ) : (
-                                <Paragraph
-                                    id={notMatchMessageId.current}
-                                    className="ffe-screenreader-only"
-                                >
-                                    {getNotMatchText()}
-                                </Paragraph>
+                                state.listToRender.length === 0 && (
+                                    <Paragraph
+                                        id={notMatchMessageId.current}
+                                        className="ffe-screenreader-only"
+                                    >
+                                        {getNotMatchText()}
+                                    </Paragraph>
+                                )
                             )}
                         </>
                     )}
                     <div id={listBoxRef.current} role="listbox">
                         {state.isExpanded &&
                             state.listToRender.map((item, index) => {
+                                const key = Object.keys(item)
```

Comment:
Vi kan bruke Object.values og det er vell ikke noen poeng med og fjerne spaces i dette tillfelel? `'' !== ' '`
```
const key = Object.values(item)
                                    .join('-')
```

Reply:
Jeg bruker `Object.keys` siden det er støttet av IE11. Nettbanken skal ikke lenger støtte IE11, men jeg tenker at designsystemet like så greit kan gjøre det hvis det er så enkelt som i dette tilfellet.

Jeg fjerner spaces for å være på den sikre siden. Jeg er redd for at en key `hei på deg` kanskje kan skape noen bugs som en key `hei-på-deg` ikke gjør. #safetyfirst :construction_worker:

Reply:
Hva med å bruke JSON.stingify(item) for å genere key?

Reply:
Det burde kunne funka.  Yter det like bra/dålig?

Reply:
Har ikke benchmark på det. Men på den måte får man en key med mindre risk å få en false positive på == enn å kun se på values.